vsan/types/types.go

```diff
@@ -1199,7 +1199,7 @@ func init() {
 type VsanPerfQueryPerfRequestType struct {
 	This       types.ManagedObjectReference  `xml:"_this"`
 	QuerySpecs []VsanPerfQuerySpec           `xml:"querySpecs"`
-	Cluster    *types.ManagedObjectReference `xml:"cluster,omitempty"`
+	Cluster    types.ManagedObjectReference `xml:"cluster,omitempty"`
```

Comment:
The `Cluster` param is optional and this struct is generated from the API wsdl. Rather than change this, you can pass a pointer as the param (e.g. `&cluster` above)

Reply:
I had incorporated the feedback in and kept it as a original. Can you please check again and approve pull request.